spec/lib/degenerate_spec.rb

```diff
@@ -33,11 +33,11 @@
 
   describe "any generator" do
     generative do
-      data(:any) { generate(:array) }
+      data(:any) { generate(:any) }
 
       it "registers an any generator" do
         class_name = any.class.to_s.downcase.to_sym
-        expect(Degenerate::Generators::GENERATORS)
+        expect(Degenerate::Generators::GENERATORS.map{|g| g == :integer ? :fixnum : g})
```

Comment:
Line is too long. [87/80]<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.